manifest.json

```diff
@@ -44,6 +44,6 @@
     "release": "v1.1.0-alpha.1"
   },
   "cli": {
-    "tag": "v1.0.2"
+    "tag": "namespace-isolation"
```

Comment:
May want to merge https://github.com/hyperledger/firefly-cli/pull/195 first and tag a release

Reply:
Yep. Good idea.

Reply:
https://github.com/hyperledger/firefly-cli/releases/tag/v1.1.0-alpha.1